selfdrive/ui/qt/util.cc

```diff
@@ -256,3 +256,9 @@ QPixmap bootstrapPixmap(const QString &id) {
   }
   return pixmap;
 }
+
+bool hasExperimentalLongitudinal(const cereal::CarParams::Reader &car_params) {
```

Comment:
```suggestion
bool hasLongitudinalControl(const cereal::CarParams::Reader &car_params) {
```

this function is to return if openpilot will control longitudinal without needing to restart to get a new carparams, think this name is more appropriate